frontend_lib/src/helper.js

```diff
@@ -857,6 +857,18 @@ const seekUsernameEnd = (text, offset) => {
   return offset
 }
 
+export const tinymceRemove = (selector) => {
+  try {
+    globalThis.tinymce.remove(selector)
+  } catch (e) {
+    if (e instanceof TypeError) {
+      console.error('HACK: removing TinyMCE raised a TypeError exception. If the message looks like "Can\'t access dead object", this probably means TinyMCE has been removed too late, or several times. Ignoring the exception but please fix this.', e)
```

Comment:
It will be nice to create an issue for this

Reply:
done :-)